Modelica/Magnetic/FluxTubes/Material/SoftMagnetic/Steel/Steel_9SMn28K.mo

```diff
@@ -1,12 +1,14 @@
 within Modelica.Magnetic.FluxTubes.Material.SoftMagnetic.Steel;
 record Steel_9SMn28K "9SMn28k (1.0715)"
   extends FluxTubes.Material.SoftMagnetic.BaseData(
+    label="9SMn28k",
```

Comment:
```suggestion
record Steel_9SMn28K "9SMn28K (1.0715)"
  extends FluxTubes.Material.SoftMagnetic.BaseData(
    label="9SMn28K",
```

The are both spellings around, with `k` and `K`. I am following the record name here...